server/spoonacularUtils/SpoonacularAPI.ts

```diff
@@ -1,4 +1,4 @@
-import axios, { AxiosResponse } from "axios";
```

Comment:
Can we add a TODO to add rate limiting by IP or something like that? So that we don't let one person take up all the requests?

Reply:
We already done that

Reply:
SpoonacularUtils folder contains SpoonacularAPI base class. This class does all heavy lifting necessary for figuring out rate limit and such.

Reply:
> Can we add a TODO to add rate limiting by IP or something like that? So that we don't let one person take up all the requests?

just reread your question... you meant by IP... so that's hard because IPs are dynamic... most of the time so we would need to have like different table and stuff on MongoDB. You can add it to issues but it will be tier 2 at best